packages/inference/src/tasks/nlp/featureExtraction.ts

```diff
@@ -2,6 +2,8 @@ import { InferenceOutputError } from "../../lib/InferenceOutputError";
 import type { BaseArgs, Options } from "../../types";
 import { request } from "../custom/request";
 
+const HF_INFERENCE_API_PIPELINE_FEATURE_EXTRACTION = "feature-extraction";
```

Comment:
The constant is only used once / not worth exporting, can you hardcode it below and remove the declaration here?

(maybe later we'll type `pipeline` as `"feature-extraction" | ... | string` to offer some kind of completion)

Same remark for the two other tasks